prow/plugins.yaml

```diff
@@ -429,37 +429,29 @@ plugins:
   - yuks
 
   kubernetes-incubator:
+  - approve
   - assign
+  - blunderbuss
+  - cat
   - cla
+  - dog
+  - heart
+  - help
+  - hold
   - label
   - lgtm
   - lifecycle
+  - shrug
   - size
-
-  kubernetes-incubator/bootkube:
-  - approve
-  - blunderbuss
-  - hold
+  - skip
   - wip
-
-  kubernetes-incubator/kubespray:
-  - approve
-  - blunderbuss
+  - yuks
 
   kubernetes-incubator/service-catalog:
-  - approve
-  - blunderbuss
-  - cat
-  - dog
-  - help
-  - hold
   - trigger
   - verify-owners
-  - wip
```

Comment:
Leave `wip` enabled like it is for `bootkube`?

Reply:
the diff is weird,  moved wip to be enabled at the org level

Reply:
🤦‍♂️